File: VB/11_Conversion/ToWord.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Pdf
Imports Spire.Pdf.Graphics

Namespace ToWord
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            'pdf file
            Dim file As String = "..\..\..\..\..\..\Data\Sample5.pdf"

            'open pdf document
            Dim doc As New PdfDocument()
            doc.LoadFromFile(file)

            'convert to xps file.
            doc.SaveToFile("Sample5.doc", FileFormat.DOC)
            doc.Close()

            'Launching the xps file.
            System.Diagnostics.Process.Start("Sample5.doc")
        End Sub
    End Class
End Namespace

File: CS/11_Conversion/ToWord.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToWord
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //pdf file
            String file = @"..\..\..\..\..\..\Data\Sample5.pdf";

            //open pdf document
            PdfDocument doc = new PdfDocument();
            doc.LoadFromFile(file);

            //convert to xps file.
            doc.SaveToFile("Sample5.doc", FileFormat.DOC);
            doc.Close();

            //Launching the xps file.
            System.Diagnostics.Process.Start("Sample5.doc");
        }
    }
}
